app/src/sharedTest/java/com/nextlevelofandroiddevelopment/marvelgallery/helpers/MainViewHelper.kt

```diff
@@ -0,0 +1,20 @@
+package com.nextlevelofandroiddevelopment.marvelgallery.helpers
+
+import com.naxtlevelofandroiddevelopment.marvelgallery.model.MarvelCharacter
+import com.naxtlevelofandroiddevelopment.marvelgallery.view.main.MainView
+
+class MainViewHelper(
```

Comment:
it is hard to reason what this class does based on name and logic, plus *Helper suffix indicates it is an util class and utils are evil:)